### ISSUE TITLE
Editing the Registry doesn't work properly

### ISSUE THREAD

**Issue by Jepl4r** [italomandara/CXPatcher#92]
I'm trying to play Horizon Zero Dawn and I'm modifying the DWORD value to 0x0000044c (1100) to remove the slow motion bug, but any edits I make in the registry don't get saved. (I already tried with a new bottle)

I don't know if this is an issue derived from CrossOver being patched or not. 

Is there any alternate method of modifying that value?

**Comment by italomandara** [comment]
It's a known issue but not sure if it's CXPatcher causing it

**Comment by Jepl4r** [comment]
Is there a workaround? Can the value be added to one of the .reg files inside the bottle folder?

**Comment by italomandara** [comment]
> Is there a workaround? Can the value be added to one of the .reg files inside the bottle folder?

I was able to set it by trying several times

**Comment by Jepl4r** [comment]
Directly inside the Registry?

**Comment by Jepl4r** [comment]
I know that. I was asking if you did that in the registry, which you did, or you modified the .reg files in the bottle

**Comment by italomandara** [comment]
https://github.com/Gcenx/WineskinServer/issues/424#issuecomment-1619239181

**Comment by Jepl4r** [comment]
@italomandara That works. I created a text file, copied the command with the right game path, saved it and changed the extension to .bat
I then created a shortcut in CrossOver. Surprisingly it starts from the shortcut directly (before I needed to use Run With Options otherwise it wouldn't start)

**Comment by Jepl4r** [comment]
@italomandara I discovered with surprise that creating a .bat file with the command to start an exe file starts all the games that previously could not be started with the shortcut created by CrossOver, for now I used it to start some GOG games that I had installed with the offline installer, so I don't know if it works with Steam games.

**Comment by italomandara** [comment]
In theory in your batch file you start steam and that's all you need

**Comment by Jepl4r** [comment]
I'm wondering if this could work to start the game using the Steam Game ID

```
@echo off
steam steam://rungameid/{GameID}
```

**Comment by Jepl4r** [comment]
> This isn’t a registry editing issue as that registry key is updated by wine itself on launch that’s why it changes. The reason the workaround works is your importing a reg file _after_ wines already running so it won’t touch that key again until it’s next launch.

Yeah, I figured. I actually tried editing the registry key and then opened the game without closing the registry and it worked. I was asking if there was a method to make it save that. Anyway that command @italomandara sent does that automatically.
